sass/components/Form/dropdown-group.scss

```diff
@@ -0,0 +1,3 @@
+.kubevirt-dropdown-group .dropdown-menu {
```

Comment:
I would prefer to call this `.kubevirt-dropdown__dropdown-group .dropdown-menu` and move it to `sass/components/Form/dropdown.scss`

Reply:
`sass/components/Form/dropdown.scss` contains styles for the `DropdownButton` component.

`sass/components/Form/dropdown-group.scss` contains styles for the `ButtonGroup` component, which wraps the `DropdownButton` component.

```jsx
<ButtonGroup className="kubevirt-dropdown-group">
  <DropdownButton className="kubevirt-dropdown">
    {/* stuff */}
  </DropdownButton>
</ButtonGroup>
```

The motivation behind this was to impose style granularity per logical React component .

However, looking at `sass/components/Form/form-group.scss` there's some redundancy - notice the `width: 100%` rule:

```css
.kubevirt-form-group .dropdown > .dropdown-menu {
  width: 100%;
  display: none;
  z-index: 1001;
}
```

I'll fix this redundancy by moving the `width: 100%` rule into `sass/components/Form/dropdown.scss` as suggested.